src/software/new_geom/vector_test.cpp

```diff
@@ -81,8 +81,7 @@ TEST(VectorLogicTests, norm_vector_from_vector_with_length_test)
 
 TEST(VectorLogicTests, norm_near_zero_vector_test)
 {
-    Vector u =
-        Vector(GeomConstants::FIXED_EPSILON * 0.99, GeomConstants::FIXED_EPSILON * 0.99);
+    Vector u = Vector(::FIXED_EPSILON * 0.99, FIXED_EPSILON * 0.99);
```

Comment:
```suggestion
    Vector u = Vector(FIXED_EPSILON * 0.99, FIXED_EPSILON * 0.99);
```

Reply:
what the heck how am I missing these. Fixed